civis/compat.py

```diff
@@ -0,0 +1,14 @@
+# flake8: noqa
+
+import six
+
+if six.PY3:
+    from unittest import mock
+    from functools import lru_cache
+    from inspect import signature
+    FileNotFoundError = FileNotFoundError
+else:
+    import mock
```

Comment:
`mock` is only a requirement for development, but this will error if it's not installed. How about
```python
try:
    import mock
except ImportError:
    pass
```
or else add a `civis/compat_tests.py` module.